app/cdap/components/hydrator/components/TopPanel/ActionButtons/styles.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import styled from 'styled-components';
+import { Button, Tooltip, withStyles } from '@material-ui/core';
+import IconSVG from 'components/shared/IconSVG';
+
+export const ActionButtonsContainer = styled.div`
+  display: inline-block;
+  min-width: 460px;
+  position: relative;
+  text-align: initial;
+  vertical-align: middle;
+`;
+
+export const BaseButton = styled(Button)`
+  background-color: white;
```

Comment:
Do we have theme values for any of the colors in this file? Unsure but we should check

Reply:
Just checked, it's just `white`

Reply:
What about the other colors? Did you check in `colors.scss`/`colors.ts`?

Reply:
Oh I misunderstood Sean's comment. Will check